### PR TITLE
Fix match delete, astros age, and lowercase email problem

### LINKED ISSUE #81: :bug: Lowercase login
If i capitalize my email address during login, it comes back as invalid.

### LINKED ISSUE #82: 🐛 Matches delete not working
Tried out the match delete, and it's not working. Pretty sure it's because the delete has a body on it. 🤔 

Try changing the endpoint to: 
`/matches/:dog_id` 
And delete things that way. ^_^

Error info: 
Screen Shot 2022-10-08 at 8 39 07 PM
Screen Shot 2022-10-08 at 8 41 34 PM
Screen Shot 2022-10-08 at 8 41 42 PM

### LINKED ISSUE #83: Astro doesn't have an age
Screen Shot 2022-10-09 at 6 30 48 PM

### PR DESCRIPTION
**Pull request by jonfinger**
Closes: #83 , #81 , #82 

#83: Astro now has an age. Other dog's ages are more appropriate as well
image

#81: emails are now lowercased always. `Jon@sniffr.be == jon@sniffr.be` now
image

#82: You can now delete a match using `/matches/int:dog_id` to delete a match. The route has been updated in postman
image

### CODE REVIEW

File: seed_db.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def seed_db_dog():
     db.session.add(
         Dog(
             dog_name="Max",
-            age="15",
+            age="5",
```
Comment (allie-rae) [inline seed_db.py:136]: Max actually lived to 15 years old :3

File: seed_db.py [hunk 2]
```diff
@@ -161,7 +161,7 @@ def seed_db_dog():
     db.session.add(
         Dog(
             dog_name="Cerberus",
-            age="10",
+            age="1000",
```
Comment (allie-rae) [inline seed_db.py:164]: lolol

File: seed_db.py [hunk 3]
```diff
@@ -147,7 +147,7 @@ def seed_db_dog():
     db.session.add(
         Dog(
             dog_name="Siri",
-            age="10",
+            age="8",
```
Comment (allie-rae) [inline seed_db.py:150]: Siri is 10 years old XD you weirdo

**Review by allie-rae — APPROVED** [review]
we're sooooooooo close!

File: seed_db.py (hunk 1 above)
Comment (jonfinger) [reply seed_db.py:136]: ohhh hahaha i thought i just put a random number pr something.  we can put that back :D

File: seed_db.py (hunk 3 above)
Comment (jonfinger) [reply seed_db.py:150]: omg im ridiculous

**Pull request merged**